### PR TITLE
New 'streaming export' ng doc

### PR DESCRIPTION
**Pull request by zuluecho9**
NR-55229

### CODE REVIEW

**Review by akristen — DISMISSED** [review]
Looks good! Nothing sticks out -- am wondering if we should default to collapsers for NerdGraph in general, or if audience typically prefers it to be laid out like this.

**Comment by zuluecho9** [comment]
> Looks good! Nothing sticks out -- am wondering if we should default to collapsers for NerdGraph in general, or if audience typically prefers it to be laid out like this.

my general preference is to not use collapsers, esp as I think coders like to skim quickly through docs. we've gotten complaint in possible that too much collapsers make things hard to parse. i avoid them mainly except when it's really a case of a set of options being mutually exclusive options
